docs/user-guide/mvd-configuration.md

```diff
@@ -353,33 +413,85 @@ TTLSCipherParms                   cipher~ZSS
 }
 ```
 
-#### Configuring the Zowe App Server for HTTPS communication with ZSS
+#### Configuring the Zowe App Server for AT-TLS communication with ZSS
 Copy the CA certificate to the ZSS server. Then in the Zowe App Server configuration file, specify the location of the certificate, and add a parameter to specify that ZSS uses AT-TLS.
 
-1. Enter the following command to copy the CA certificate to the correct location in UNIX System Services (USS):
+1. Put the CA certificate ZSS uses into a location the App server can access, such as a keyring or a secure location in UNIX System Services (USS), such as the keystore:
 
 ```
-cp "//'[output_dataset_name]'" '[INSTANCE_DIR]/workspace/app-server/serverConfig/[ca_cert]'
+cp "//'[output_dataset_name]'" '[KEYSTORE_DIRECTORY]/[ca_cert]'
 ```
-2. In the `[INSTANCE_DIR]/workspace/app-server/serverConfig` directory, open the `server.json` file.
-3. In the **node.https.certificateAuthorities** object, add the CA certificate file path, for example:
+2. In the `[INSTANCE_DIR]` directory, open the `instance.env` or `zowe.yaml` file.
+3a. If using instance.env, define `ZWED_node_https_certificateAuthorities` equal to a comma-separated list of CA files and keyring references that satisfy all servers the app-server will talk to, meaning not only ZSS but also the rest of Zowe.
+3b. If using zowe.yaml, define components.app-server.node.https.certificateAuthorities equal to a comma-separated list of CA files and keyring references that satisfy all servers the app-server will talk to, meaning not only ZSS but also the rest of Zowe.
```

Comment:
I vaguely remember having some issues with special chars, are those accounted for in doc?